### ISSUE TITLE
Kick Stool

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#193]
thinking of getting one of these to reach the top shelves in the garage:

https://www.leroymerlin.pt/Produtos/Ferragens/Escadotes-tamboretes/Tamboretes/WPR_REF_82440760

![16A59B38-A356-4016-9A86-1E66913DFCBC](https://user-images.githubusercontent.com/194400/157777729-3f2f75c7-d0bd-49d4-ae65-94d74f302384.png)

using the step ladder for now. 💭

**Comment by nelsonic** [comment]
Wish there was a more _attractive_ version of this made from Wood even if it was like €100 ...

**Comment by nelsonic** [comment]
Out of stock in Braga:
![image](https://user-images.githubusercontent.com/194400/166134699-5718ac06-f6e0-464a-8019-b9c9320b132d.png)

So can't get it today. Might have to order online.

**Comment by iteles** [comment]
Should we look for something that folds away and isn't constantly going to be underfoot in the kitchen?

If you have just 3 people milling around in there, this will be in the way the second someone pulls it out.

**Comment by nelsonic** [comment]
@iteles dunno if you saw the first line in the OP. but I wan this for the _Garage/Workshop_ not the kitchen. 🧰 
It could be _borrowed_ for the kitchen, but definitely wouldn't _live_ in the kitchen. 💭 
We need something else to use in the kitchen. Separate issue. 🪜

**Comment by iteles** [comment]
Ooooh sorry, I'm in kitchen mode 😅 One thing to add to the projecto!

**Comment by nelsonic** [comment]
@iteles yeah, if we go ahead with a "projecto" with Leroy then defo add it to get the discount. 💡 
Should I be adding the tools I want too? 😜
